### PR TITLE
Minor tweaks for CI tests

### PR DESCRIPTION
**Pull request by aakarshg**
No wait is required if we're waiting for job to complete
Added a small wait for 5 seconds after deploying operator
Cleaning up benchmark-operator before deleting cb operator

### CODE REVIEW

**Comment by aakarshg** [comment]
@dustinblack PTAL

File: tests/test_couchbase.sh [hunk 1]
```diff
@@ -6,8 +6,8 @@ source tests/common.sh
 function finish {
   echo "Cleaning up couchbase"
   kubectl delete -f tests/test_crs/valid_couchbase.yaml
-  kubectl delete deployment couchbase-operator
   delete_operator
+  kubectl delete deployment couchbase-operator
```
Comment (dustinblack) [inline tests/test_couchbase.sh]: I think this delete deployment may only need to be here at all in the case of a failed or aborted run. Anecdotally, it seems like it gets cleaned up properly without this line if the job finishes properly.

**Comment by dustinblack** [comment]
Small comment, but it's not exactly related to this commit. Otherwise LGTM.

File: tests/test_couchbase.sh (hunk 1 above)
Comment (aakarshg) [reply tests/test_couchbase.sh]: yep that's right if the job finishes properly it gets cleaned up automatically, I think in that case I can probably add a check to see if the deployment is present and delete only if. Good catch @dustinblack

**Comment by aakarshg** [comment]
Closing this and will work on a new PR since #69 got merged

**Pull request closed without merging**